src/Elmah/XmlFileErrorLog.cs

```diff
@@ -78,6 +79,20 @@ public XmlFileErrorLog(IDictionary config)
                 logPath = MapPath(logPath);
 
             _logPath = logPath;
+
+
+            var fileListSizString = config.Find("size", string.Empty);
```

Comment:
`fileListSizString` appears to have a typo; should read `fileListSizeString` instead.